test/integration/http_transport_errors_test.py

```diff
@@ -0,0 +1,83 @@
+import os
+import time
+import pytest
+import shutil
+import pyexasol
+
+
+@pytest.fixture
+def dev_null():
+    with open(os.devnull, "wb") as f:
+        yield f
+
+
+@pytest.fixture
+def statement():
+    yield "SELECT * FROM USERS LIMIT 1000"
+
+
+@pytest.fixture
+def invalid_statement():
+    yield "SELECT * FROM DoesNotExist LIMIT 1000"
+
+
+@pytest.mark.etl
+@pytest.mark.exceptions
+def test_sql_error(connection, dev_null, invalid_statement):
+    def export_cb(pipe, dst, **kwargs):
+        shutil.copyfileobj(pipe, dev_null)
+
+    with pytest.raises(pyexasol.exceptions.ExaQueryError):
```

Comment:
Maybe also assert exception message?

Reply:
In my opinion, there's occasional merit in verifying the actual Exception message, alongside the current test. Be aware, though, this could lead to a more fragile test. In this instance, I didn't see the need to confirm the actual exception message. Thanks for pointing it out though :+1: .